modules/Components/src/client/PickList.coffee

```diff
@@ -685,6 +706,7 @@ class EditablePickListSelect2Controller extends EditablePickListSelectController
 			collection: @collection
 			selectedCode: @options.selectedCode
 			filters: filters
+			autoFetch: @options.autoFetch
```

Comment:
*sort of outside pr scope*: Pass options auto fetch to the picklist controller options as noted elsewhere in my review.